Gemfile.lock

```diff
@@ -217,6 +217,7 @@ GEM
 PLATFORMS
   arm64-darwin-21
   arm64-darwin-22
+  arm64-darwin-23
```

Comment:
I acknowledge this change. I would like an explanation of why NOT this should be added to the gem file. I think from research this specifies the supported platforms. Seeing as I just built the project on said platform, I reckon it is supported. Let me know and I can rip this one out.